yb-voyager/cmd/importDataStatusCommand.go

```diff
@@ -178,18 +293,42 @@ func prepareImportDataStatusTable() ([]*tableMigStatusOutputRow, error) {
 			status = "MIGRATING"
 		}
 		row := &tableMigStatusOutputRow{
-			fileName:           path.Base(dataFile.FilePath),
-			tableName:          dataFile.TableName,
-			status:             status,
-			totalCount:         totalCount,
-			importedCount:      importedCount,
-			percentageComplete: perc,
+			tableName:  dataFile.TableName,
+			schemaName: getTargetSchemaName(dataFile.TableName),
+		}
+		if withStreamingMode {
+			qualifiedTableName := qualifyTableName(tconf.Schema, row.tableName)
+			eventCounter, err := tdb.GetImportedEventsStatsForTable(qualifiedTableName, migrationUUID)
+			if err != nil {
+				return nil, fmt.Errorf("get imported events stats for table %q: %w", qualifiedTableName, err)
+			}
+			row.totalEvents = eventCounter.TotalEvents
+			row.numInserts = eventCounter.NumInserts
+			row.numUpdates = eventCounter.NumUpdates
+			row.numDeletes = eventCounter.NumDeletes
+			row.importedCount, err = state.GetImportedRowCount(dataFile.FilePath, dataFile.TableName)
+			if err != nil {
+				return nil, fmt.Errorf("get imported row count for table %q: %w", qualifiedTableName, err)
```

Comment:
You won't need this once you move the code in the `else` block back in the row object instantiation.